terra-jupyter-base/Dockerfile

```diff
@@ -117,6 +118,11 @@ RUN git clone https://github.com/broadinstitute/cromshell.git \
 # verify cromshell 2.0 is installed
 RUN cromshell-alpha version
 
+# copy workspace_cromwell.py script
+RUN git clone https://github.com/broadinstitute/cromwhelm.git \
+ && cd cromwhelm \
+ && cp scripts/workspace_cromwell.py $HOME
```

Comment:
It seems simpler (but there may be other downsides) to just fetch the latest version of the file directly

```
wget https://raw.githubusercontent.com/broadinstitute/cromwhelm/main/scripts/workspace_cromwell.py
```

Reply:
This sounds like a good idea. We don't need anything else but the script. Do you know what downsides would be for this approach?

Reply:
I don't know any off the top of my head, just leaving the door open to the possibility you specifically chose `git clone` with something in mind

Reply:
Upside is we don't have a leftover `cromwhelm` checkout hanging around, which _I think_ may get bundled into the final image and slightly increase its size

Reply:
I see. I had actually forgotten about `wget`. And I think it will be a better approach instead of cloning the whole repo.